recipes/schwimmbad/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "schwimmbad" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "e2574edb3c25b59d480001cdd77a73b6808340cad8c2a288fb1b85614277967d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - six
+    - mpi4py
+
+test:
+  imports:
+    - schwimmbad
+
+about:
+  home: http://github.com/adrn/schwimmbad
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'A common interface for parallel processing pools'
+
+  description: |
+    schwimmbad enables easy switching between serial execution, multiprocessing,
+    and MPI by providing a common interface.
+  doc_url: https://github.com/adrn/schwimmbad
+  dev_url: https://github.com/adrn/schwimmbad
+
+extra:
+  recipe-maintainers:
+    - guillochon
+    - pkgw
+    - adrn
```

Comment:
@adrn - are you ok being added as a co-maintainer of the `schwimmbad` recipe on conda-forge?